php/libraries/NDB_BVL_Instrument.class.inc

```diff
@@ -1391,7 +1406,7 @@ class NDB_BVL_Instrument extends NDB_Page
      *
      * @return Integer The sessionID
      */
-    function getSessionID()
+    function getSessionID(): Integer
```

Comment:
integer or bool (false, line 1419)

Reply:
I am going to change this to return `-1` instead. SessionID in other classes should always be an int going forward.